share/completions/tmutil.fish

```diff
@@ -1,61 +1,136 @@
+complete -c tmutil -f -n '__fish_seen_subcommand_from compare' -s u -d 'Compare UIDs'
+complete -c tmutil -r -n '__fish_seen_subcommand_from compare' -s D -d 'Limit traversal depth to depth levels from the beginning of iteration'
+complete -c tmutil -f -n '__fish_seen_subcommand_from compare' -s E -d 'Dont take exclusions into account'
+complete -c tmutil -r -n '__fish_seen_subcommand_from compare' -s I -d 'Ignore path'
+complete -c tmutil -f -n '__fish_seen_subcommand_from compare' -s U -d 'Ignore logical volume identity'
+
+complete -c tmutil -r -n __fish_use_subcommand -a delete -d 'Delete one or more snapshots'
+complete -c tmutil -r -n '__fish_seen_subcommand_from delete' -s d -d 'Backup mount point'
+complete -c tmutil -r -n '__fish_seen_subcommand_from delete' -s t -d 'Timestamp'
+complete -c tmutil -r -n '__fish_seen_subcommand_from delete' -s p -d 'Path'
+
+complete -c tmutil -r -n __fish_use_subcommand -a deletelocalsnapshots -d 'Delete all local Time Machine snapshots for the specified date (formatted YYYY-MM-DD-HHMMSS)'
+
+complete -c tmutil -r -n __fish_use_subcommand -a deleteinprogress -d 'Delete all in-progress backups for a machine directory'
+
+complete -c tmutil -f -n __fish_use_subcommand -a destinationinfo -d 'Print information about destinations'
+
+complete -c tmutil -f -n __fish_use_subcommand -a disable -d 'Turn off automatic backups'
+
+complete -c tmutil -f -n __fish_use_subcommand -a disablelocal -d 'Turn off local Time Machine snapshots'
+
+complete -c tmutil -f -n __fish_use_subcommand -a enable -d 'Turn on automatic backups'
+
+complete -c tmutil -f -n __fish_use_subcommand -a enablelocal -d 'Turn on local Time Machine snapshots'
+
+complete -c tmutil -r -n __fish_use_subcommand -a inheritbackup -d 'Claim a machine directory or sparsebundle for use by the current machine'
+
+complete -c tmutil -F -n __fish_use_subcommand -a isexcluded -d 'Determine if a file, directory, or volume are excluded from backups'
+
+complete -c tmutil -r -n __fish_use_subcommand -a latestbackup -d 'Print the path to the latest snapshot'
+
+complete -c tmutil -f -n __fish_use_subcommand -a listlocalsnapshotdates -d 'List the creation dates of all local Time Machine snapshots'
+
+complete -c tmutil -r -n __fish_use_subcommand -a listlocalsnapshots -d 'List local Time Machine snapshots of the specified volume'
+
+complete -c tmutil -f -n __fish_use_subcommand -a listbackups -d 'Print paths for all snapshots'
+
+complete -c tmutil -f -n __fish_use_subcommand -a localsnapshot -d 'Create new local Time Machine snapshot of APFS volume in TM backup'
+
+complete -c tmutil -f -n __fish_use_subcommand -a machinedirectory -d 'Print the path to the current machine directory'
+
+complete -c tmutil -f -n __fish_use_subcommand -a removedestination -d 'Removes a backup destination'
+complete -c tmutil -f -n '__fish_seen_subcommand_from removedestination' -d "$(__destination_ids)"
```

Comment:
You want this to be an *argument*, not a description, presumably? I.e. `-a` instead of `-d`?

Reply:
Yes. I corrected one of those but I missed one. Thank you for catching it.